src/olympia/addons/serializers.py

```diff
@@ -421,7 +421,9 @@ def fake_object(self, data):
                     display_username=persona_data['author'],
                     header=persona_data['header'],
                     footer=persona_data['footer'],
-                    persona_id=1 if persona_data['is_new'] else None,
+                    # "New" Persona do not have a persona_id, it's a relic from
+                    # old ones.
+                    persona_id=0 if persona_data['is_new'] else 42,
```

Comment:
`1` was not remarkable enough, `42` denotes it's a hack :) Yeah, converting the files would be a much better solution I agree.